### PR TITLE
Read me

### PR DESCRIPTION
**Pull request by UGcode101**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,113 @@
...
+2. **Create a Playlist**: Select three genres to create a playlist.
+
+3. **Block Songs/Artists**: Users can block specific songs or artists from appearing in their playlists via the settings.
+
+4. **Manage Preferences**: User preferences are saved and can be managed through the profile section.
+
+## Technologies Used
+
+- **Java & Spring Boot**: Backend framework
+- **Thymeleaf**: Templating engine
```
Comment (JenniferNedland) [inline README.md:85]: We did not use thymeleaf.

File: README.md [hunk 2]
```diff
@@ -1 +1,113 @@
...
+- **MySQL**: Database
+- **Spotify Web API**: Integration with Spotify
+- **React**: Frontend framework (if applicable)
+- **Grid: CSS**: Layout and design
+
+## Contributing
```
Comment (JenniferNedland) [inline README.md:91]: Do we want to invite contributions and pull requests from unknown users?

**Review by JenniferNedland — COMMENTED** [review]
Thanks for putting this together! Added a few minor changes and one question.

File: README.md [hunk 3]
```diff
@@ -1 +1,113 @@
...
+4. **Build and Run the Application**
+
+   Use Gradle to build and run the application:
+
+   ```bash
+   ./gradlew build
+   ./gradlew bootRun
+   ```
+
+5. **Access the Application**
```
[the remark this answers is not in the document]
Comment (UGcode101) [reply README.md:68]: fixed

**Comment by UGcode101** [comment]
resolved all comments

File: README.md [hunk 4]
```diff
@@ -48,26 +47,26 @@ ThreeMix is a web application that allows users to create custom playlists by se
 
 3. **Configure Spotify API**
 
-   Update your `application.properties` with your Spotify API credentials:
+ Create a clientConstants class and put the secret in the folder. This folder is not checked in to Github.
```
Comment (JenniferNedland) [inline README.md:50]: `ClientConstants` should be Pascal cased.

File: README.md [hunk 5]
```diff
@@ -48,26 +47,26 @@ ThreeMix is a web application that allows users to create custom playlists by se
...
 
    ```properties
-   spotify.client.id=your-client-id
-   spotify.client.secret=your-client-secret
-   spotify.redirect.uri=http://localhost:8080/callback
+package org.launchcode.threemix.secret;
+
+public class ClientConstants {
+public static final String CLIENT_ID = "";
```
Comment (JenniferNedland) [inline README.md:56]: Line 56 and 57 should keep their correct indentations.

File: README.md [hunk 6]
```diff
@@ -48,26 +47,26 @@ ThreeMix is a web application that allows users to create custom playlists by se
...
+public static final String CLIENT_SECRET = "";
+}
    ```
 
 4. **Build and Run the Application**
 
    Use Gradle to build and run the application:
 
-   ```bash
```
Comment (JenniferNedland) [inline README.md]: The backend also needs to be started, so keep this part too.

**Review by JenniferNedland — DISMISSED** [review]

**Review by JenniferNedland — APPROVED** [review]

**Review by JenniferNedland — APPROVED** [review]

**Review by JenniferNedland — DISMISSED** [review]
cleaned it up thanks!

**Review by JenniferNedland — DISMISSED** [review]

**Pull request merged**